Source/IGListAdapterUpdater.m

```diff
@@ -23,7 +23,7 @@ @implementation IGListAdapterUpdater
 
 - (instancetype)init {
     IGAssertMainThread();
-
+    
```

Comment:
Mind going through and stripping all this whitespace?

Reply:
Happy to do that :)